docs/user-guide/street-view.md

```diff
@@ -0,0 +1,26 @@
+# Street View
+*******************
+
+The **Street View** tool allows the user to browse [Google Street View](https://www.google.com/streetview/) contents in MapStore. Through the <img src="../img/button/street-view-button.jpg" class="ms-docbutton"/> button available in the *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, the tool can be activated so that it is possible to navigate the map with [Google Street View](https://www.google.com/streetview/). 
+
+<img src="../img/street-view/streetview-popup.jpg" class="ms-docimage" width="500px"/>
+
+When the tool is activated, a window opens and the streets highlighted on the map so that the user can select one of them with a simple click of the mouse.
+
+<img src="../img/street-view/add-street.gif" class="ms-docimage"/>
+
+Clicking on a street in map the tool's window visualize the Street View and the user can navigate it as usual.
```

Comment:
By clicking on a street in the map, the tool window displays the Street View and the user can navigate it as usual.